libr/debug/p/native/linux/linux_debug.c

```diff
@@ -97,6 +97,68 @@ int linux_handle_signals(RDebug *dbg, int tid) {
 		// siginfo.si_code -> HWBKPT, USER, KERNEL or WHAT
 		// TODO: DO MORE RDEBUGREASON HERE
 		switch (dbg->reason.signum) {
+		case SIGABRT: // 6 / SIGIOT // SIGABRT
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
```

Comment:
set this before the switch. and only handle the ones that are not signals

Reply:
Oh my god ! I had only handled signals in this whole commit.

Reply:
Keep thing simple